indico/web/client/js/react/util/Nexus.jsx

```diff
@@ -0,0 +1,62 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2023 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React, {useCallback, useState} from 'react';
+import {Portal} from 'semantic-ui-react';
+
+/**
+ * HOC to make out-of-tree components look like they are placed exactly over another
+ * component. This is useful e.g. to escape CSS cascading rules.
+ *
+ * @param {Node} children Children elements
+ * @param {Node} target Target element
+ * @param {Object} overrides Style overrides to be applied on the placeholder
+ * @param {boolean} open Whether the component is visible
+ */
+export default function Nexus({children, target, overrides, open}) {
+  const [style, setStyle] = useState(null);
+  const placeholderCallback = useCallback(
+    node => {
+      if (node && open) {
+        const rect = node.getBoundingClientRect();
```

Comment:
I think we somehow need to refresh this when showing the message that saving succeeded... IntersectionObserver etc. look incredibly complex: https://stackoverflow.com/q/40251082/298479

![image](https://github.com/indico/indico/assets/179599/ff3b0420-4d80-407b-8e21-54281e9e8554)